src/Skia/Avalonia.Skia/PlatformRenderInterface.cs

```diff
@@ -145,6 +145,8 @@ public IRenderTarget CreateRenderTarget(IEnumerable<object> surfaces)
             {
                 if (surface is IFramebufferPlatformSurface framebufferSurface)
                     return new FramebufferRenderTarget(framebufferSurface);
+                if (surface is IExternalCanvasSurface canvas)
```

Comment:
@kekekeks some api like this?
`IPlatformRenderInterface.CreateRenderTarget` with interface for Canvas and Dpi

Reply:
Anything in AvaloniaLocator is technically a private API. So I'd rather have some class with a `DrawingContext WrapSkiaCanvas(Canvas canvas, Vector dpi)` static method in `Avalonia.Skia` assembly.